.github/actions/palletjack/deploy/action.yml

```diff
@@ -28,7 +28,7 @@ runs:
 
     - name: 🚀 Deploy to Cloud Function
       id: deploy
-      uses: google-github-actions/deploy-cloud-functions@v1
+      uses: google-github-actions/deploy-cloud-functions@v2
       with:
         name: roadkill-skid
         runtime: python39
```

Comment:
**thought**: updating python could maybe help.

Reply:
I wanted to leave that can of worms for dbot day.